sdk/core/perf-tests/core-rest-pipeline/test/node-fetch.spec.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+
+import fetch from "node-fetch";
```

Comment:
is the issue that global fetch is conflicting with the name? In `core-http` we did `import node_fetch from "node-fetch";`

Reply:
That doesn't look like the issue.
Fails with
```bash
Error [ERR_REQUIRE_ESM]: require() of ES Module /workspaces/azure-sdk-for-js/common/temp/node_modules/.pnpm/node-fetch@3.2.3/node_modules/node-fetch/src/index.js from /workspaces/azure-sdk-for-js/sdk/core/perf-tests/core-rest-pipeline/dist-esm/node-fetch.spec.js not supported.
Instead change the require of index.js in /workspaces/azure - sdk -for-js / sdk / core / perf - tests / core - rest - pipeline / dist - esm / node - fetch.spec.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous>(/workspaces/azure - sdk -for-js / sdk / core / perf - tests / core - rest - pipeline / dist - esm / node - fetch.spec.js: 7: 46)
    at Object.<anonymous>(/workspaces/azure - sdk -for-js / sdk / core / perf - tests / core - rest - pipeline / dist - esm / index.spec.js: 8: 27) {
  code: 'ERR_REQUIRE_ESM'
```

Reply:
Deleted the test for now. Logging an issue